kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -89,7 +90,7 @@ public KafkaProxy(Configuration config) {
      * @return This proxy.
      */
     public KafkaProxy startup() throws InterruptedException {
-        LOGGER.info("Kroxylicious starting");
+        STARTUP_SHUTDOWN_LOGGER.info("Kroxylicious starting");
```

Comment:
I want an Asciiart kroxy as a startup message.

Reply:
does https://github.com/k-wall/kroxylicious/pull/3 fit the bill?

Reply:
Adopted :)